include/usgscsm/Distortion.h

```diff
@@ -6,22 +6,24 @@
 #include <tuple>
 #include <vector>
 
-enum DistortionType { RADIAL, TRANSVERSE, KAGUYALISM, DAWNFC, LROLROCNAC, CAHVOR };
+// This should be synched with the enum in ale/Distortion.h
+enum DistortionType { RADIAL, TRANSVERSE, KAGUYALISM, DAWNFC, LROLROCNAC, CAHVOR, 
+                     LUNARORBITER, RADTAN };
 
-// Transverse Distortion
-void distortionJacobian(double x, double y, double *jacobian,
-                        const std::vector<double> opticalDistCoeffs);
+// Transverse distortion Jacobian
+void transverseDistortionJacobian(double x, double y, double *jacobian,
+                                  std::vector<double> const& opticalDistCoeffs);
 
 void computeTransverseDistortion(double ux, double uy, double &dx, double &dy,
-                                 const std::vector<double> opticalDistCoeffs);
+                                 std::vector<double> const& opticalDistCoeffs);
 
 void removeDistortion(double dx, double dy, double &ux, double &uy,
-                      const std::vector<double> opticalDistCoeffs,
+                      std::vector<double> const& opticalDistCoeffs,
                       DistortionType distortionType,
                       const double tolerance = 1.0E-6);
 
 void applyDistortion(double ux, double uy, double &dx, double &dy,
-                     const std::vector<double> opticalDistCoeffs,
+                     std::vector<double> const& opticalDistCoeffs,
                      DistortionType distortionType,
                      const double desiredPrecision = 1.0E-6,
                      const double tolerance = 1.0E-6);
```

Comment:
I am a little concerned here as this is API breaking, know we haven't started enforcing that in USGSCSM but we likely should. @Kelvinrr @jlaura Thoughts?

Reply:
Thoughts on this are very welcome. I can easily undo this interface change. I assumed it was fine because these are internal functions, not mean to be called outside of usgscsm I think.

Reply:
I rebased to latest and fixed a conflict

Reply:
I dont think this counts as API breaking? iirc const-ness doesn't change if it's in before or after the type. And passing by reference wound't change how a user interacts with the function.